app/javascript/lib/metrics-api.js

```diff
@@ -330,6 +331,22 @@ function getAnalyticsPageName() {
   return getSCPContext().analyticsPageName
 }
 
+/**
+ * gets the tab name for analytics
+ */
+function getTabProperty() {
```

Comment:
Non-blocking: While this totally works, it seems overly complicated.  Using `window.location.hash` will always return the open tab as we have JS that will append this to the URL on every click.  In the case of the upload wizard, I think there's a simpler way to get the info.  For the final case, `window.location.hash` returns a string, so the `else` clause will cover that as well and return `''`.
```suggestion
function getTabProperty() {
  if (window.location.href.match(/\?step=/)) {
    return window.location.href.split('?step=')[1]
  } else {
    return window.location.hash.replace(/#/, '')
  }
}
```

Reply:
Oh nice! much cleaner!